src/main/app/case/case.ts

```diff
@@ -92,6 +92,7 @@ export interface Case {
   anyNameChangeSinceRelationshipFormed?: YesOrNo;
   changedNameHow?: ChangedNameHow;
   changedNameHowAnotherWay?: string;
+  respondentEmailAddress?: string;
```

Comment:
how comes this is added here? is it something missed from another ticket as i dont see it being needed or used for this ticket.

Reply:
Ah, this is needed for the the dynamic content:

```ts
formState?.respondentEmailAddress ? 'the email address you entered earlier' : 'email, text message or social media'
```

However, it since this field hasn't been implemented yet, we'll have to remember to double check that this field name is still correct once the ticket for their partners email has been merged.